site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -35,6 +35,9 @@ link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas
 ==== Request Body
 
 [.api]
+[field]#displayName# [type]#[String]# [optional]#Optional#::
+A human readable name for this link to help you identify this link. This value will generally be an email address, or username. This value is recorded when the link is made, and updated during the next login for this identity provider. This value may not necessarily reflect the username or email you used to authenticate with the 3rd party identity provider
```

Comment:
I don't understand what this means:

> This value is recorded when the link is made, and updated during the next login for this identity provider.

Reply:
Needs a period at the end.

Reply:
Can you give an example: `richard@piedpiper.com`? 

Is this a way to track/search all the links for a given user?

Reply:
This is a convenience field.  Updated wording to better clarify.